### ISSUE TITLE
Image Annotation | Giving users custom annotation capabilities

### ISSUE THREAD

**Issue by bpugh5** [Soundbendor/smart-bin-app#10]
# User Story

As a Smart Compost Bin User, I want to be able to specify my own bounding boxes and annotations for each bin reading so that I can contribute to the formation of a more accurate dataset.

### Why?

This feature will allow users to easily create and modify the digital rectangles and annotation notes that specify their composted items in just a few steps. With a more intuitive system, users will feel more motivated to contribute their revisions.
### Who?

This feature is for the user.
### Can we support it?

The application should be able to support it. It will require usage of some external packages, but can definitely be done.
## Solution/Requirements

- When a new image is taken by a bin, an entry should appear in the app.
- The user should be able to click on the entry image and view it
- The user should have access to a digital toolbar that allows them to add/remove bounding boxes, as well as annotation text boxes for the items.

## Dependencies

- The smart bin must be capable of taking photos of user-contributed compost items
- Blocked by lack of other app functionality
- The bounding boxes must be adjustable in order to make the dataset contributions as accurate as possible

## Estimate

- Initial design - 2 weeks
- Proof of concept - 3 weeks

- Complete implementation - 5 weeks
## When this Issue is Done

This Issue is complete once:

- Users can click on a recently completed scan and open the annotation menu.
- The user can click on a bounding box create button, and grow/shrink it to highlight the item(s) they just put into the bin.
- The user can click on an annotation create button, and attach it to any of the bounding boxes to provide a description of what the bounding boxes are highlighting.
- Required verifications from team members are made and relevant pull requests/commits are merged into dev/main.
